<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BDDCourseData</class>
 <widget class="QWidget" name="BDDCourseData">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="5">
    <widget class="QLabel" name="m_nbr5">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="m_discipline">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="m_nbr4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QLabel" name="m_place2Label">
     <property name="text">
      <string>Place 2: </string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="m_place1Label">
     <property name="text">
      <string>Place 1:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="7">
    <widget class="QLabel" name="m_place1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="m_disciplineLabel">
     <property name="text">
      <string>Discipline:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="m_distanceLabel">
     <property name="text">
      <string>Distance:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="m_nbr3">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="m_nbr6">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QLabel" name="m_winner">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="m_nbr7">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="m_nbr1">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="m_winnerLabel">
     <property name="text">
      <string>Gagnant:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="7">
    <widget class="QLabel" name="m_place2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="m_nbrLeaverLabel">
     <property name="text">
      <string>Nombre partant:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="m_nbrLeaver">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="9">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="m_distance">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="m_nbr2">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <widget class="QLabel" name="m_nbr8">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="m_horsesLabel">
     <property name="text">
      <string>Nombres :</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="3">
    <widget class="QTableView" name="m_en3"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QTableView" name="m_en2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="6" colspan="3">
    <widget class="QTableView" name="m_en2En3">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>310</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
